pkg/compute/compute_test.go

```diff
@@ -30,19 +30,81 @@ func TestInit(t *testing.T) {
 		t.Skip("Set TEST_COMPUTE_INIT to run this test")
 	}
 
-	if err := os.Chdir("testdata/init"); err != nil {
-		t.Fatal(err)
-	}
-
 	for _, testcase := range []struct {
-		name       string
-		args       []string
-		wantError  string
-		wantOutput string
+		name            string
+		args            []string
+		wantFiles       []string
+		unwantedFiles   []string
+		wantError       string
+		wantOutput      []string
+		manifestContent string
```

Comment:
Since this is a strings.Contains rather than a full equality comparison, minor suggestion to rename to manifestContains or manifestIncludes?